lms/static/js/ccx/schedule.js

```diff
@@ -203,7 +214,7 @@ var edx = edx || {};
 	    self.schedule_collection.set(self.schedule);
 	    var button = $('#dirty-schedule #save-changes');
 	    button.prop('disabled', true).text(gettext("Saving")+'...');
```

Comment:
I know this is outside of your changes but could you remove the '...' from this string? It is not compatible with internationalization (or I believe accessibility) and we've worked to remove these from strings. Obviously this one is concatenated so I missed it.